java/openmldb-batch/src/test/scala/com/_4paradigm/openmldb/batch/catalog/TestOpenmldbCatalogService.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 4Paradigm
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com._4paradigm.openmldb.batch.catalog
+
+import org.scalatest.{FunSuite, Ignore}
+
+@Ignore
+class TestOpenmldbCatalogService extends FunSuite {
+
+  test("TestOpenmldbCatalogService") {
+    val zkCluster = "127.0.0.1:2181"
```

Comment:
Why ignore this test? cicd java-sdk job can run it

Reply:
It required external ZooKeeper cluster to run. Developers may compile this projects without setting up ZK cluster or using another config.

We may use in-process ZK mini cluster to test the functions which requires ZK.

Reply:
mini cluster needs zk too, it'll only replace the openmldb servers.
We have a lot of java tests which need zk. Maybe we should apply one rule to them.